### PR TITLE
Add whatsnew page for firefox 73 (Fixes #8436)

### LINKED ISSUE #8436: WNP73: Code
## Description

In #8428 we'll build several campaign pages. This card is about coding WNP73, which will be a primary entry point for the campaign. 

* WNP73 will take the copy from the campaign landing page
* It will have 3 content items below the main CTA area; [WNP70 experiment v5](﻿﻿https://docs.google.com/presentation/d/1PlWlpWK3L27W2zmZ2Jp1VVhbDwUGMPohHCOEIQRl-wM/edit#slide=id.g7c4f53cf2e_0_1222) is an approach that could work. 
* it will have a state-based CTA with two states. IF Firefox is the default, the CTA will be the mobile send-to-device widget. If Firefox is not the default, the CTA will encourage users to set Firefox as the default browser. The set-default CTA will link to the campaign's thanks page.

Designs for the campaign landing page (https://github.com/mozilla/bedrock/issues/8429) will be very helpful in building the WNP73. 

[Copy doc for WNP73](https://docs.google.com/document/d/1LzZfDKW-UK-QcD3qaY59jXiwiNZTmr2adcr2-2VejV0/edit#)

[Releng bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1609993)

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] WNP73 is launched in production by 2020/01/23
- [ ] All approvals complete
- [ ] Ready for staff localization in Tier 1
- [ ] Strings extracted for community localization

#### Approvals
- [ ] RASCI A
- [ ] PM

### PR DESCRIPTION
**Pull request by alexgibson**
## Description

[Copy doc for WNP73](https://docs.google.com/document/d/1LzZfDKW-UK-QcD3qaY59jXiwiNZTmr2adcr2-2VejV0/edit#)

[Design](https://github.com/mozilla/bedrock/issues/8442)

## Issue / Bugzilla link
#8436

## Testing
Demo URL: https://www-demo5.allizom.org/en-US/firefox/73.0/whatsnew/all/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx73.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+          <img src="{{ static('img/firefox/mobile/protocol/qr-firefox.png') }}"
+               id="firefox-qr"
+               data-mozillaonline-link="{{ static('img/firefox/mobile/protocol/qr-firefox-mozillaonline.png') }}"
+               alt="{{ _('QR code to scan for Firefox') }}">
+        </div>
+        {% endif %}
+
+        <ul class="mobile-download-buttons" id="mobile-download-buttons-firefox">
+          <li>
+            {{ google_play_button(href=android_url, id='playStoreLink') }}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx73.html]: `android_url` is undefined here

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx73.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+          </li>
+          <li>
+            <a id="appStoreLink" href="{{ ios_url }}" data-link-type="download" data-download-os="iOS">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx73.html]: `ios_url` is undefined

File: bedrock/firefox/views.py [hunk 3]
```diff
@@ -588,6 +588,8 @@ def get_template_names(self):
                 template = 'firefox/whatsnew/index.html'
         elif locale == 'id':
             template = 'firefox/whatsnew/index-lite.id.html'
+        elif version.startswith('73.'):
```
Comment (alexgibson) [inline bedrock/firefox/views.py]:
```suggestion
        elif version.startswith('73.') and lang_file_is_active('firefox/whatsnew_73', locale):
```

File: tests/functional/firefox/whatsnew/test_whatsnew_73.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from selenium.common.exceptions import TimeoutException
+from pages.firefox.whatsnew.whatsnew_73 import FirefoxWhatsNew73Page
+
+
+@pytest.mark.skip_if_not_firefox(reason='Whatsnew pages are shown to Firefox only.')
```
Comment (alexgibson) [inline tests/functional/firefox/whatsnew/test_whatsnew_73.py]: We should replace these tests with a single test for the default CTA, which doesn't depend on Firefox being set as the default browser by the OS.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx73.html [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-l-content">
+      <aside id="modal" class="mzp-u-modal-content mzp-l-content">
+        <h2 class="modal-logo">Firefox</h2>
+        <h3>{{ _('Get Firefox on your Phone') }}</h3>
+
+        {% if show_send_to_device %}
+          {{ send_to_device(include_title=False, message_set='fx-mobile-download-desktop', class='vertical') }}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx73.html]: I'm guessing this should be `fx-whatsnew`
```suggestion
          {{ send_to_device(include_title=False, message_set='fx-whatsnew', class='vertical') }}
```

**Comment by alexgibson** [comment]
Successful test run: https://gitlab.com/mozmeao/www-config/pipelines/110779654

File: bedrock/firefox/views.py [hunk 6]
```diff
@@ -588,7 +588,7 @@ def get_template_names(self):
                 template = 'firefox/whatsnew/index.html'
         elif locale == 'id':
             template = 'firefox/whatsnew/index-lite.id.html'
-        elif version.startswith('73.'):
+        elif version.startswith('73.') and lang_file_is_active('firefox/whatsnew_73', locale):
```
Comment (stephaniehobson) [inline bedrock/firefox/views.py]: 🔥

**Comment by stephaniehobson** [comment]
@alexgibson I'll leave this for you to merge because I am not sure about the interdependancies but it looks good to me.

**Pull request merged**